pages/common/date.md

```diff
@@ -26,3 +26,7 @@
 - Display the current date using the RFC-3339 format (`YYYY-MM-DD hh:mm:ss TZ`):
 
 `date --rfc-3339=s`
+
+- Set the current date using the format `MMDDhhmm[YYYY][.ss]`:
+
+`date {{09302359}}`
```

Comment:
@indigane Could you please apply this suggestion from here? https://github.com/tldr-pages/tldr/pull/7019#discussion_r740158395
```suggestion
- Set the current date using the format `MMDDhhmm`:

`date {{09302359}}`

- Set the current date using the format `MMDDhhmmYYYY.ss`:

`date {{093023592021.59}}`
```

Reply:
I'm sorry, but I contributed this suggestion, because it was something I needed myself, and the suggestion is no longer useful to me. I realize this is a bit selfish, and you are all welcome to take it from here.

These two suggested examples are misleading, they remove too much information. Both year and seconds are optional.

This also seems to be against the spirit of tldr; "Try to keep pages at around 5 examples."

The single example is both more concise and more correct.

I understand that the second guideline also makes a note to keep newer command line users in mind, but I feel this is too far into misleading that I can't support it.

Reply:
Don't worry about being selfish, almost all of our pages exist, because someone found them useful.

Regarding the examples, I don't understand why they're misleading. Both commands work and describe what they do. But I think there's another solution. How about this?
```suggestion
- Set the current date using the format `MMDDhhmmYYYY.ss` (`YYYY` and `.ss` are optional):

`date {{093023592021.59}}`
```

Reply:
I think we can fix that too. Some suggestions:
* ``Both `YYYY` and `.ss` are optional``
* `` `YYYY` and/or `.ss` can be omitted``